fmatch/matcher.py

```diff
@@ -166,6 +166,43 @@ def filter_runs(self, pdata, data):
         ids_df = ndf.loc[df['jobConfig.jobIterations'] == iterations]
         return ids_df['uuid'].to_list()
 
+    def getResults(self, uuid: str, uuids: list, index_str: str, metrics: dict):
+        """
+        Get results of elasticsearch data query based on uuid(s) and defined metrics
+
+        Args:
+            uuid (str): _description_
+            uuids (list): _description_
+            index_str (str): _description_
+            metrics (dict): _description_
+
+        Returns:
+            dict: Resulting data from query
+        """
+        if len(uuids) > 1 and uuid in uuids:
+            uuids.remove(uuid)
+        ids = '" OR uuid: "'.join(uuids)
+        metric_string = ""
+        for k, v in metrics.items():
+            if k == "not":
+                for not_list in v:
+                    for k1, v1 in not_list.items():
+                        # f' AND NOT (jobConfig.name: "garbage-collection")'
+                        if isinstance(v1,str):
+                            v1 = f'"{v1}"'
+                        metric_string += f" AND NOT ({k1}: {v1})"
+            elif isinstance(v,str) and not k in ['metric_of_interest']:
```

Comment:
Can we please add ```'metric'``` here instead of popping in the orion PR. This way I believe we can use it if we need it, and not lose data. same goes to ```get_agg_metric_query()```

Reply:
yes will do